pyroute2/dhcp/timers.py

```diff
@@ -49,7 +47,7 @@ def arm(self, lease: Lease, **callbacks: AsyncCallback) -> None:
         loop = asyncio.get_running_loop()
 
         for timer_name, async_callback in callbacks.items():
-            self._reset_timer(timer_name)  # type: ignore[arg-type]
+            self._reset_timer(timer_name)
```

Comment:
Not sure if it is a good idea to add some CI test that raises warnings if there are any inline linter overrides.

Reply:
as long as it does not make the CI fail, i see no issue with it !